beacon/aeon_details.go

```diff
@@ -244,3 +290,14 @@ func loadAeonDetailsFiles(filePath string) ([]*AeonDetailsFile, error) {
 
 	return aeonQueue, err
 }
+
+// Function to return whether the specified validator is in qual
+func (aeon *aeonDetails) HasValidatorInQual(addr types.Address) bool {
+
+	if aeon.validators == nil {
+		return false
+	}
+
+	return aeon.validators.HasAddress(addr)
+}
```

Comment:
This does not check if the validator is in qual as `aeon.Validators` contains all validators that ran the DKG. You need to use the validator index and check `aeon.aeonExecUnit.InQual(index)`